src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertificate/HealthCertificateViewModel.swift

```diff
@@ -309,13 +309,10 @@ final class HealthCertificateViewModel {
 			topSpace: 2.0
 		)
 
-		var dateOfBirthCellViewModel: HealthCertificateKeyValueCellViewModel?
-		if let dateOfBirthDate = healthCertificate.dateOfBirthDate {
-			dateOfBirthCellViewModel = HealthCertificateKeyValueCellViewModel(
-				key: "Geburtsdatum / Date of Birth (YYYY-MM-DD)",
-				value: ISO8601DateFormatter.justLocalDateFormatter.string(from: dateOfBirthDate)
-			)
-		}
+		let dateOfBirthCellViewModel = HealthCertificateKeyValueCellViewModel(
+			key: "Geburtsdatum / Date of Birth (YYYY-MM-DD)",
+			value: DCCDateStringFormatter.formatedString(from: healthCertificate.dateOfBirth)
+		)
```

Comment:
I think we need to also apply this to the date of vaccination and the date of birth in the vaccination details (and the recovery details, but they are not merged yet: #2963)

Reply:
Right. I'll create a task.

Reply:
https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-7988

Reply:
"I think we need to also apply this to the date of vaccination and the date of birth in the vaccination details"

Done!